plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/ILibraryClasspathContainerResolverService.java

```diff
@@ -32,25 +33,39 @@
   public static final String LIBRARIES_EXTENSION_POINT =
       "com.google.cloud.tools.eclipse.appengine.libraries"; //$NON-NLS-1$
 
+  enum Runtime {
+    AppEngineStandard
```

Comment:
OTOH, "Runtime" means something different in Eclipse. I.e. the word is overloaded. Maybe rename AppEngineRuntime? and the enum would just be Standard?